package.json

```diff
@@ -27,7 +27,7 @@
 		"flatpickr": "^4.5.2"
 	},
 	"peerDependencies": {
-		"svelte": "^3.31.0"
+		"svelte": ">= 3.31.0 < 4.1"
```

Comment:
Does this not work with `svelte >= 4.1`, or what was the reason for the restriction?